validator/modules/onnx/__init__.py

```
@@ -0,0 +1,289 @@
+from typing import List, Optional
+from pydantic import BaseModel
+from validator.exceptions import RecoverableException
+from validator.modules.base import (
+    BaseValidationModule,
+    BaseConfig,
+    BaseInputData,
+    BaseMetrics,
+)
+from huggingface_hub import hf_hub_download
+import onnxruntime as ort
+import os
```

Ruff (0.12.2)

`os` imported but unused

(F401)